.github/workflows/test_report.yml

```diff
@@ -26,6 +26,7 @@ on:
 
 jobs:
   test_report:
+    if: github.event.workflow_run.conclusion != 'skipped'
```

Comment:
queustion: I only saw `github.event.workflow_run.conclusion` in [link](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow), `skipped` is not mentioend in here, but looks like it works from your test. Any reference?

Reply:
The `workflow_run` payload is not well documented: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_run

I have collected a few workflow_run events and investigated the `conclusion` field.

Reply:
OK, thanks for invistigation, does the `skiped` means all jobs skipped?

What if part of jobs passed/failed and left part of jobs skipped?

Reply:
Tested here: https://github.com/EnricoMi/spark/actions/runs/2234527115
Triggered: https://github.com/EnricoMi/spark/actions/runs/2234869750
![grafik](https://user-images.githubusercontent.com/44700269/165598727-5260e893-a43b-4a72-ae31-0c2983c447ef.png)

Reply:
Awesome, thanks!